0011-interledger-payment-request/0011-interledger-payment-request.md

```diff
@@ -99,42 +62,17 @@ Additional binary headers, encoded as a base64 string, that should be included i
 
 It is RECOMMENDED that recipients use a Message Authentication Code (MAC) of the details of their payment request in the condition so that the condition can be used as an integrity check on incoming transfers. This allows a recipient to ensure that a transfer matches a request they previously generated, without the recipient needing to maintain a log of all outstanding payment requests.
 
-Recipients MUST include a payment request-specific component in the account ILP Address to ensure that conditions are unique per-request.
-
-### Preimage Condition from JSON
+Recipients MUST include a payment request-specific nonce in the ILP packet to ensure that conditions are unique per-request.
```

Comment:
This sounds like something I have to worry about in addition to using PSK, but doesn't using PSK take care of this?